sdk/tests/deqp/functional/gles3/es3fFramebufferBlitTests.js

```diff
@@ -726,7 +726,7 @@ goog.scope(function() {
         for (var ndx = 0; ndx < 2; ndx++) {
             ctx.bindFramebuffer(gl.FRAMEBUFFER, ndx ? dstFbo : srcFbo);
             if (ndx) {
-                gradientToDstShader.setGradient(ctx, gradShaderDstID, dstRangeInfo.valueMax, dstRangeInfo.valueMin);
+                gradientToDstShader.setGradient(ctx, gradShaderDstID, dstRangeInfo.valueMin, dstRangeInfo.valueMax);
                 rrUtil.drawQuad(ctx, gradShaderDstID, [-1, -1, 0], [1, 1, 0]);
             } else {
                 gradientToSrcShader.setGradient(ctx, gradShaderSrcID, srcRangeInfo.valueMin, srcRangeInfo.valueMax);
```

Comment:
That's true, the C++ code is not correct here. It should be srcRangeInfo.valueMin for the last parameter on that line @qkmiao pointed out. Because the dstRangeInfo.valueMax is different from srcRangleInfo.valueMax, they are defined here: https://android.googlesource.com/platform/external/deqp/+/master/framework/common/tcuTextureUtil.cpp#355 

Seems that the c++ code is buggy at this code snippet. @zhenyao